svn/common.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, url_or_path, type_, *args, **kwargs):
         self.__trust_cert = kwargs.pop('trust_cert', None)
 
         if type_ not in (svn.constants.LT_URL, svn.constants.LT_PATH):
-            raise ValueError("Type is invalid: %s" % (type_))
+            raise SvnException("Type is invalid: %s" % type_)
```

Comment:
I was thinking of having SvnException as the parent exception of all the exception's raised by  PySvn. @dsoprea What do you think about this? According to me, we should have custom exceptions only and the solution here is to update the docstring for providing more apt information.